webapp/src/ts/modules/reports/reports.component.ts

```diff
@@ -191,7 +209,17 @@ export class ReportsComponent implements OnInit, OnDestroy {
   };
 
   search(force = false) {
-    // todo filters
+    // clears report selection for any text search or filter selection
+    // does not clear selection when someone is editing a form
+    if((this.filters.search || Object.keys(this.filters).length > 1) && !this.enketoEdited) {
+      //$state.go('reports.detail', { id: null }, { notify: false });
```

Comment:
Do we still need this "if"?

Reply:
Yea, I think so. It's still commented because we don't have a report detail page yet. :D